### DISCUSSION TITLE
Package.Current equivalent in Winui3?

### DISCUSSION THREAD

**Discussion by christopher-carmichael** [microsoft/microsoft-ui-xaml#8467] [Q&A] [answered]
I am trying to include the app version in my settings page, however I cannot seem to find a way to access the package information in winui3. Does anyone know a work around?

**Comment by DarranRowe** [comment, marked as answer]
Use Package.Current?
WinUI 3 is just a library running in a regular desktop process, it isn't special. Desktop applications have no issues using Windows.ApplicationModel.Package. So just remember, you have the entire desktop application API available to you.
